games-arcade/rocksndiamonds/additional-files/rocksndiamonds.rdef.in

```diff
@@ -0,0 +1,38 @@
+// Rocks 'n' Diamonds icon by Stephanie Fu
+
+resource app_signature "application/x-vnd.Artsoft-rocksndiamonds";
+
+resource app_flags B_EXCLUSIVE_LAUNCH;
+
+resource app_version {
+    major  = @MAJOR@,
+    middle = @MIDDLE@,
+    minor  = @MINOR@,
+    variety = B_APPV_DEVELOPMENT,
+    internal = @INTERNAL@,
+    short_info = "Rocks'n'Diamonds",
+    long_info = "Rocks'n'Diamonds, an arcade style game.\nCopyright 2001-2013 Artsoft Entertainment"
```

Comment:
No. Unprintable chars are chars that:
* you don't see with
```
cat rocksndiamonds.rdef.in
```
* and that you see with
```
cat -v rocksndiamonds.rdef.in
```
These can be _control chars_ like, e.g. the famous Control-G which is a bell that you can hear when you type
```
echo ^G
```
(to obtain the ^G you have to press Control-V and then Control-G)

On your file you have unbreakable spaces (0xA0), as well as another unprintable char (0xC2).

Poor editors won't show them to you.
vi does show all unprintable chars, except tabs, which are "renedered" if we may say so.